packages/cli/test/functional/test_site_algolia_plugin/index.md

```diff
@@ -30,12 +30,12 @@
 
 **Popover content should have algolia-no-index class**
 
-<popover effect="fade" header="Title" placement="top">
+<popover header="Title" placement="top">
```

Comment:
I believe you accidentally included the changes for the test and template files in this PR 😅

Reply:
Sorry I missed it previously, I've made the changes!